frontend/src/pages/PipelineDetails.tsx

```diff
@@ -296,21 +296,18 @@ class PipelineDetails extends Page<{}, PipelineDetailsState> {
         );
 
         if (listVersionsResponse.pipeline_versions) {
```

Comment:
We should probably also check that `len(listVersionsResponse.pipeline_versions) > 0`